sweepai/core/vector_db.py

```diff
@@ -316,6 +338,52 @@ def compute_deeplake_vs(collection_name, documents, ids, metadatas, sha):
         return deeplake_vs
 
 
+def compute_embeddings(documents):
```

Comment:
I don't understand: why did it create this function? It's not used anywhere in this PR and none of the code seems to be existing from before.